server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -611,16 +616,19 @@ func (s *BuildBuddyServer) GetBazelConfig(ctx context.Context, req *bzpb.GetBaze
 	if eventsAPIURL == "" {
 		eventsAPIURL = assembleURL(req.Host, "grpc:", grpcPort)
 	}
-	groupAPIKey := s.getGroupAPIKey(ctx)
-	eventsAPIURL = insertPassword(eventsAPIURL, groupAPIKey)
+	groupAPIKeys, err := s.getAPIKeysForAuthorizedGroup(ctx)
+	if err != nil {
+		return nil, err
+	}
+	eventsAPIURL = insertPassword(eventsAPIURL, "${API_KEY}")
 	configOptions = append(configOptions, makeConfigOption("build", "bes_backend", eventsAPIURL))
 
 	if s.env.GetCache() != nil {
 		cacheAPIURL := s.env.GetConfigurator().GetAppCacheAPIURL()
 		if cacheAPIURL == "" {
 			cacheAPIURL = assembleURL(req.Host, "grpc:", grpcPort)
 		}
-		cacheAPIURL = insertPassword(cacheAPIURL, groupAPIKey)
+		cacheAPIURL = insertPassword(cacheAPIURL, "${API_KEY}")
```

Comment:
can we just get rid of this insert password stuff altogether now that we're not shoving the api keys into the urls. don't think we need this ${API_KEY} stuff